### ISSUE TITLE
Additions to Document model

### ISSUE THREAD

**Issue by opoudjis** [riboseinc/asciidoctor-iso#24]
These will need be deferred until after #23. 

* Termnotes need to have anchors. This will also help with labelling.
* Need multiple footnote references for the same footnote text. Asciidoc supports this, and the Rice document needs it.
* Inline subsection numbers in test methods and other such clauses in annexes.

**Comment by opoudjis** [comment]
Left alignment on paragraphs.

Introduce thematic breaks (page breaks)

**Comment by opoudjis** [comment]
It turns out pagebreak is in the isoxml code, but not in the document model yet.

**Comment by opoudjis** [comment]
Document review_notes need to be dated (and date added to comment attributes in Word)

**Comment by opoudjis** [comment]
Table will need option count of header rows and footer rows, where they are more than one: #36.

**Comment by opoudjis** [comment]
Make term definition optional.

**Comment by opoudjis** [comment]
Add iteration of recirculated draft to document header.

**Comment by opoudjis** [comment]
Left alignment on paragraphs: done. #35.

**Comment by opoudjis** [comment]
Remaining items:

* Termnotes need to have anchors. This will also help with labelling.
* Need multiple footnote references for the same footnote text. Asciidoc supports this, and the Rice document needs it.
* Inline subsection numbers in test methods and other such clauses in annexes.
* Table will need option count of header rows and footer rows, where they are more than one: #36.
* Make term definition optional.
* Add iteration of recirculated draft to document header.

**Comment by opoudjis** [comment]
Make these separate tickets.
